File: problems/intermediate/organizing_container/__tests__/test_organizing_container.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from organizing_containers import organizingContainers

def test_organizing_containers():
    containers = [[1, 4], [2, 3]]
    result = organizingContainers(containers)
    assert result == "Impossible", f"Expected Impossible, got {result}"

    print("✅ Python test passed!")

if __name__ == "__main__":
    test_organizing_containers()

File: problems/intermediate/organizing_container/__tests__/TestOrganizingContainer.java
package problems.intermediate.organizing_container.__tests__;

import problems.intermediate.organizing_container.OrganizingContainer;
import java.util.*;


public class TestOrganizingContainer {
    public static void main(String[] args) {
        List<List<Integer>> containers = Arrays.asList(
            Arrays.asList(1, 4),
            Arrays.asList(2, 3)
        );

        String result = OrganizingContainer.organizingContainers(containers);

        if (!result.equals("Impossible")) {
            throw new AssertionError("❌ Test failed. Expected Impossible, got " + result);
        }

        System.out.println("✅ Java test passed!");
    }
}
